docker/docker-compose-dev.yml

```diff
@@ -41,21 +43,8 @@ services:
     ]
     ports:
         - "6379:6379"
+    networks:
+    -   actinia-dev
 
-
-  # fluentd:
-  #   build:
-  #     context: fluentd
-  #   image: fluentd
-  #   expose:
-  #     - 24224
-  #
-  # elasticsearch:
-  #   image:  elasticsearch:7.4.0
-  #   environment:
-  #   - discovery.type=single-node
-  #
-  # kibana:
-  #   image: kibana:7.4.0
-  #   ports:
-  #     - "5601:5601"
+networks:
+  actinia-dev:
```

Comment:
Just for my understanding: Here we define the network such that both `actinia` and `redis` can be in the same network? But as it is here, no further configuration of the network is required?

Reply:
Correct, when nothing is specified, all services inside the compose file are using the same network on startup.
Here a docker network is specified explicitly. When redis is started, it will automatically be created and used without further configuration. When actinia is started via debugger, this network is also specified: https://github.com/mundialis/actinia_core/blob/909d0cd41e14862fbea72e14504c2200c64b674a/.vscode/tasks.json#L44 
Docker compose names the network like `<name-of-folder-where-compose-file-is-inside>_<name-of-network>` ).